# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../apmd/X11R7.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- apmd/Makefile.vanilla	2005-10-09 18:43:02.000000000 +0200
+++ apmd/Makefile	2005-10-09 18:43:23.000000000 +0200
@@ -46,7 +46,7 @@
 		-DAPMD_PROXY_NAME=\"$(APMD_PROXY_DIR)/apmd_proxy\"
 LDFLAGS=
 XTRALDFLAGS=-L. -lapm
-XLDFLAGS=-L/usr/X11R6/lib
+XLDFLAGS=-L/usr/X11/lib
 XLIBS=-lXaw -lXmu -lXt -lXext -lSM -lICE -lX11
 LIB=libapm.a
 RANLIB=ranlib
